components/raftstore-v2/src/batch/store.rs

```diff
@@ -508,6 +508,14 @@ impl<EK: KvEngine, ER: RaftEngine> StoreRouter<EK, ER> {
         &self.logger
     }
 
+    #[inline]
+    pub fn check_send(&self, addr: u64, msg: PeerMsg) -> crate::Result<()> {
```

Comment:
What does the name mean?

Reply:
Send and check the result. I will convert `Disconnected` to `RegionNotFound` error.